src/WorkflowCore/WorkflowCore.csproj

```diff
@@ -15,12 +15,12 @@
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <Description>Workflow Core is a light weight workflow engine targeting .NET Standard.</Description>
-    <Version>3.3.5</Version>
-    <AssemblyVersion>3.3.5.0</AssemblyVersion>
-    <FileVersion>3.3.5.0</FileVersion>
+    <Version>3.3.6</Version>
```

Comment:
I think we need to bump the version of the MongoDb package instead

Reply:
@danielgerlag fixed